Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\qfb_main\views.py", line 81, in news_article_list
    return render(request, 'index.html', {'NewsArticle_list': articles})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loader_tags.py", line 127, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loader_tags.py", line 124, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loader_tags.py", line 103, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loaders\cached.py", line 58, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 478, in parse
    raise self.error(token, e)
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\base.py", line 476, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ize_c\OneDrive\Desktop\Quickfire_bulletin1\venv\Lib\site-packages\django\template\loader_tags.py", line 212, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'content' appears more than once
"GET / HTTP/1.1" 500 145
